source/index.html.md

```diff
@@ -503,6 +504,72 @@ One of the reasons why SMART on FHIR is awesome is because of the interoperabili
 * You can now launch the application by clicking “launch”, and choose a patient in context (some patients with Observations include: [Allen, Carol G. | Adams, Daniel X. | Coleman, Lisa P.])
 * You will be re-directed to the OpenID connect authorization server – similar to the Cerner Millenium login screen. All you have to do is at the bottom of the screen, click “Authorize” when it asks “Do you authorize [insert app name here]”.
 
+# Standalone App Launch for Patient Access Workflow
+
+SMART supports EHR launch and standalone launch. However, Cerner currently only supports standalone launch for patient access workflow. The standalone application does not need to be launched by an EHR.  The app can launch and access FHIR data on its own, provided the app is authorized and given the iss URL.
+
+You can find out more about standalone launch on the <a href="http://docs.smarthealthit.org/authorization/" target="_blank">SMART Health IT site</a> under the "Standalone launch sequence" header.
```

Comment:
This is super nit, but for every link, I think we should just leverage normal Markdown links rather than a manual href that opens each link in a new tab. As a user, I like to control whether I open links in a new tab, new window, or directly in the current tab. I get annoyed when sites make the choice for me.